lib/components/carousal_slider.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
+import 'package:carousel_slider/carousel_slider.dart';
+import 'package:flutter/material.dart';
+
+class AppCarousel extends StatefulWidget {
+  const AppCarousel({super.key, this.height, this.width,this.imageList});
+
+  final double? height;
+  final double? width;
+  final List<String>? imageList;
+  @override
+  State<StatefulWidget> createState() => _AppCarousel();
+}
+
+class _AppCarousel extends State<AppCarousel> {
+  CarouselController carouselController = CarouselController();
+
+  final imageList = [
```

Comment:
Figure out a way to send the data from another class or page because in the future we will be getting the data from a firebase database and we will pass the data to the carousel after it has been built. Take care of some edge cases such as what if the data returned is empty.

Maybe we need a static image to display in case the data returned from the database is empty.